api/apps/api/src/modules/scenarios/scenarios.controller.ts

```diff
@@ -271,6 +271,25 @@ export class ScenariosController {
     return await this.service.getSpecDatCsv(id);
   }
 
+  @ApiTags(marxanRunFiles)
+  @ApiOperation({
+    description: `Get archived output files`,
+  })
+  @Get(`:id/marxan/output`)
+  @Header(`Content-Type`, `application/octet-stream`)
+  @Header('Content-Disposition', 'attachment; filename="output.zip"')
+  async getOutputArchive(
+    @Param(`id`, ParseUUIDPipe) scenarioId: string,
+    @Res() response: Response,
+  ) {
+    const file = await this.service.getMarxanExecutionOutputArchive(scenarioId);
+    response.set({
+      [`Content-Type`]: `application/octet-stream`,
+      [`Content-Disposition`]: `attachment; filename="output.zip"`,
+    });
```

Comment:
AFAIR it is not needed, despite having `Res` in the method, `Header` decorators still control headers.

Reply:
https://github.com/Vizzuality/marxan-cloud/pull/319/commits/e9e74a6875780002b457deec5bc20df6c0cae77e